guide/how-it-works/private-key-management/multi-key.md

```diff
@@ -97,13 +149,13 @@ A software wallet application or coordination software initiates a multi-sig wal
 - Make sure the multi-key setup itself is backed up properly, including [extended public keys]({{ '/guide/glossary/#extended-public-key-xpub-ypub-zpub' | relative_url }}) for all the participating keys, fingerprint and derivation.
 
 **Products that use this scheme**
+- [Sparrow Wallet](https://sparrowwallet.com/)
 - [Casa](https://keys.casa) co-managed 2-of-3, or 3-of-5
 - [Electrum](https://electrum.org)
-- [Bluewallet](https://bluewallet.io)
+- [BlueWallet](https://bluewallet.io)
 - [Unchained Capital/Caravan](https://unchained-capital.com) co-managed
 - [Specter](https://specter.solutions)
 - [Armory](https://btcarmory.com)
-- [Guarda](https://guarda.com)
 - [Revault](https://revault.dev){:target="_blank"} - in development
 
```

Comment:
Maybe Nunchuk would also be a nice addition?

Reply:
Added.